### ISSUE TITLE
Indication when editing sync realm offline

### ISSUE THREAD

**Issue by stel** [realm/realm-browser-osx#263]
Proposed in: https://github.com/realm/realm-browser-osx/issues/261

If the connection was lost while editing remote Realm the changes that had been made offline can potentially be lost if the Browser window is closed before the connection is available again.

In this case we can simply disable editing and show the corresponding warning because in general case it's not possible to guarantee the delivery of this changes to ROS.

/cc @bigfish24

**Comment by bigfish24** [comment]
How important do we really believe this to be? It is a desktop app?

**Comment by stel** [comment]
Yes, it's a Cocoa Browser. It's not a common case but it could cause a data loss, so it would be great to have something to prevent that.

**Comment by bigfish24** [comment]
Ok, in that case, I would not prioritize this at the moment, seems very niche.
